Zotero/Scenes/Main/Views/MainViewController.swift

```diff
@@ -111,16 +111,26 @@ final class MainViewController: UISplitViewController {
         let navigationController = UINavigationController()
         let tagFilterController = (viewControllers.first as? MasterContainerViewController)?.bottomController as? ItemsTagFilterDelegate
 
-        let coordinator = DetailCoordinator(
+        let newDetailCoordinator = DetailCoordinator(
             libraryId: libraryId,
             collection: collection,
             searchItemKeys: searchItemKeys,
             navigationController: navigationController,
             itemsTagFilterDelegate: tagFilterController,
             controllers: controllers
         )
-        coordinator.start(animated: false)
-        detailCoordinator = coordinator
+        newDetailCoordinator.start(animated: false)
+        if let detailCoordinator, presentedViewController != nil {
```

Comment:
Wouldn't this allow for a detail coordinator to include children coordinators of another library/group? Could that be an issue somewhere along the line?